packages/ibm-products-web-components/package.json

```diff
@@ -62,6 +62,7 @@
     "@ibm/telemetry-js": "^1.10.2",
     "@lit-labs/signals": "^0.1.2",
     "@lit/context": "^1.1.5",
+    "@lottiefiles/lottie-player": "^2.0.12",
```

Comment:
Can you move this to dev deps instead?

Reply:
There is also a ["light" version](https://github.com/aarsteinmedia/dotlottie-player?tab=readme-ov-file#in-javascript-or-typescript) as well which will give us even better performance (ie `import '@aarsteinmedia/dotlottie-player/light';`